src/main/scala/com/fulcrumgenomics/umi/ConsensusCaller.scala

```diff
@@ -141,6 +159,22 @@ class ConsensusCaller(errorRatePreLabeling:  PhredScore,
       val base = DnaBasesUpperCase(maxLlIndex)
       (base, q)
     }
+
+    /**
+      * Returns the calculated likelihoods of the four bases given the data and the model. The likelihoods
```

Comment:
Rather than `Returns` perhaps `Gives`.  For `def`s that have no `()` you should write their comments like they are `val`s.